app/code/VConnect/OrderVolume/Model/Order/Item/Config.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace VConnect\OrderVolume\Model\Order\Item;
+
+class Config
+{
+    public const SALES_ORDER_ITEM_TABLE = 'sales_order_item';
```

Comment:
I don't think you will need those constants if you rework your flow and will use another event for a setting volume to order and items.